orbit-2-core/src/main/java/com/babylon/orbit2/Context.kt

```diff
@@ -16,11 +16,14 @@
 
 package com.babylon.orbit2
 
+import com.babylon.orbit2.idling.IdlingResource
+
 /**
  * Represents the current context in which an [Operator] is executing.
  *
  * @property state The current state captured at the point when the operator is executed
  * @property event The current event being processed
+ * @property idlingResource TODO Write description here
  */
 @Orbit2Dsl
-data class Context<STATE : Any, EVENT>(val state: STATE, val event: EVENT)
+data class Context<STATE : Any, EVENT>(val state: STATE, val event: EVENT, val idlingResource: IdlingResource)
```

Comment:
What's the reason for having it here? In case someone wants to use it manually? Worth making it an extension property?

Reply:
left over from when I was trying to implement it in the first place I think... removed locally and code and tests pass